extensions/wikia/BannerNotifications/BannerNotifications.setup.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Docs are on internal in UI Style Guide
+ */
+$wgExtensionCredits['bannernotifications'][] = array(
+	'name' => 'BannerNotifications',
+	'descriptionmsg' => 'bannernotifications-desc',
+	'author' => [
+		'Hyun Lim',
+		'Maciej Brencz',
+		'Bartłomiej (Bart) Kowalczyk'
+	],
+	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/BannerNotifications'
+);
+
+$dir = dirname(__FILE__) . '/';
+
+/**
+ * Classes
+ */
+$wgAutoloadClasses[ 'BannerNotificationsController' ] = $dir . 'BannerNotificationsController.class.php';
+
+/**
+ * Hooks
+ */
+$wgHooks['ArticleDeleteComplete'][] = 'BannerNotificationsController::addPageDeletedConfirmation';
```

Comment:
It would be good to mention to QA that these are some back end instances to test, since it's less well know that these can come from the back end.